packages/story-editor/src/app/media/utils/useFFmpeg.js

```diff
@@ -347,19 +345,19 @@ function useFFmpeg() {
    * @return {Promise<File>} Transcoded video file object.
    */
   const stripAudioFromVideo = useCallback(
-    async (file) => {
+    async (file, fileExt) => {
       //eslint-disable-next-line @wordpress/no-unused-vars-before-return
       const trackTiming = getTimeTracker('load_mute_video_transcoding');
 
       let ffmpeg;
 
       try {
         ffmpeg = await getFFmpegInstance(file);
-
-        const tempFileName = uuidv4() + '.' + MEDIA_TRANSCODED_FILE_TYPE;
-        const outputFileName =
-          getFileName(file) + '-muted.' + MEDIA_TRANSCODED_FILE_TYPE;
-
+        const ext =
+          fileExt || getTypeFromMime(file?.type) || MEDIA_TRANSCODED_FILE_TYPE;
+        const type = file?.type || MEDIA_TRANSCODED_MIME_TYPE;
```

Comment:
If you reserve this it becomes a bit shorter:

```suggestion
        const type = file?.type || MEDIA_TRANSCODED_MIME_TYPE;
        const ext = fileExt || getTypeFromMime(type);

```

Reply:
I explained why I did this why here. https://github.com/google/web-stories-wp/pull/9031#discussion_r709221541

TDLR, if type is null for some reason, it will fallback to `MEDIA_TRANSCODED_FILE_TYPE`.

Reply:
Unless I am missing something, my code and yours do exactly the same thing:

1. Get extension from URL
2. Fall back to extension derived from `file.type`
3. Fall back to extension derived from `MEDIA_TRANSCODED_MIME_TYPE` (mp4)

My code is just shorter and easier to read IMO.

Reply:
I don't think we should be using `getTypeFromMime`. Mime !== file ext. 

Example 
- application/svg+xml - svg
- image/svg+xml - svg
- video/ogg - ogv
- video/3gpp- 3gp
- video/3gpp2 - 3g2

I know we don't support these file formats, but this just to prove that mime is not file extension. 

I think using it `getTypeFromMime` is extremely britel , if anything I would want to remove it's usage here. I like the usage of `MEDIA_TRANSCODED_FILE_TYPE` here, it is something we have more control. If we don't use `MEDIA_TRANSCODED_FILE_TYPE` let's use a function with a switch statement.

Reply:
Just had to take another look at `getTypeFromMime` to realize that it returns the _element_ type from a mime type, not the _file_ type. 🤦 

So for `video/mp4` it returns `video`. So yeah we definitely shouldn't use it, albeit for a different reason.

But my point still stands: switching the two lines avoids some repetition.

Anyhow, resolving this as it's not a big deal.